lib/aptible/api/operation.rb

```diff
@@ -41,6 +41,10 @@ def user
         nil
       end
 
+      def logs_url
+        links['logs_url']
```

Comment:
Since we made the change in API.

```suggestion
        links['logs']
```

Reply:
```
irb(main):016:0> o.links['logs'].href
=> "https://api-josh.aptible-sandbox.com/operations/383/logs"
```